core/dbt/cli/main.py

```diff
@@ -289,6 +291,52 @@ def compile(ctx, **kwargs):
     return results, success
 
 
+# dbt show
+@cli.command("show")
+@click.pass_context
+@p.defer
+@p.deprecated_defer
+@p.exclude
+@p.favor_state
+@p.deprecated_favor_state
+@p.full_refresh
+@p.show_output_format
+@p.show_limit
+@p.indirect_selection
+@p.introspect
+@p.parse_only
+@p.profile
+@p.profiles_dir
+@p.project_dir
+@p.select
+@p.selector
+@p.inline
+@p.state
+@p.deprecated_state
+@p.target
+@p.target_path
+@p.threads
+@p.vars
+@p.version_check
+@requires.preflight
+@requires.profile
+@requires.project
+@requires.runtime_config
+@requires.manifest
+def show(ctx, **kwargs):
+    """Generates executable SQL from source, model, test, and analysis files. Compiled SQL files are written to the
+    target/ directory."""
```

Comment:
This is the same help text as for `compile`. Perhaps something like:
```
    """Generates executable SQL for a named resource or inline query, runs that SQL, and returns a preview of the results."""
```
cc @dbeatty10

p.s. It is actually cool that `compile` & `show` work with `analyses`!